pkg/ddl/backfilling.go

```diff
@@ -858,30 +858,34 @@ func adjustWorkerCntAndMaxWriteSpeed(ctx context.Context, pipe *operator.AsyncPi
 			return
 		case <-ticker.C:
 			failpoint.InjectCall("onUpdateJobParam")
-			maxWriteSpeed := job.ReorgMeta.GetMaxWriteSpeed()
+			if reorgInfo != nil {
```

Comment:
Why is it checking if `the reorgInfo` is nil? It is not checked elsewhere.

Reply:
Because the upper layer  `executeAndClosePipeline` may pass `nil`, 
<img width="1112" height="215" alt="截屏2025-09-19 17 13 23" src="https://github.com/user-attachments/assets/b57f51d2-11fd-468a-9dd5-bf81869c2619" />
this goroutine is only used for local ingest add index update worker cnt.

Reply:
OOOh, i see

Reply:
update in https://github.com/pingcap/tidb/pull/63605/commits/3370636a1a9f72fe90a1ef520068e1106344a3ee, PTAL again